### DISCUSSION TITLE
https://joinmastodon.org/ns is 404

### DISCUSSION THREAD

**Discussion by UlyC** [mastodon/mastodon#22900] [Q&A] [answered]
The url  “https://joinmastodon.org/ns ” in the `@context` of json-ld  is 404. Is this expected?

``` json
{
  "@context": [
...
    {
...
      "toot": "http://joinmastodon.org/ns#",
...
   }
  ]
}
```

![image](https://user-images.githubusercontent.com/36838240/210173015-3452bb1b-9f5b-48c4-ac3f-4f37871561b5.png)

**Comment by connyduck** [comment]
apparently https://github.com/mastodon/joinmastodon/issues/148#issuecomment-1114174935

**Comment by trwnh** [comment, marked as answer]
note that it is `http://joinmastodon.org/ns#` and not `http://joinmastodon.org/ns` -- it's not a link to an external context document, it's an inline term definition. and the term definition for `toot` is only to set the base IRI for other compact terms. there is no requirement or guarantee that the IRI in a term definition resolves over http to a specification document, it's just something that would be nice.
